frontend/packages/dev-console/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -72,6 +73,8 @@ class ResourceDropdown extends React.Component<ResourceDropdownProps, State> {
       autoSelect,
     } = nextProps;
 
+    const { onLoad } = this.props;
```

Comment:
`onLoad` should be coming from `nextProps`

Reply:
Done.